### PR TITLE
Draft of TWG minutes for Nov 30

### PR DESCRIPTION
**Pull request by DBees**
Signed-off-by: DBees <duncan@openhwgroup.org>

### CODE REVIEW

**Review by jerryzeng11 — APPROVED** [review]

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Nov 30/2020
+
+## TG Reports
+
+### SW TG
+
+Written report prepared - thanks to Jeremy:
+https://github.com/openhwgroup/core-v-docs/blob/master/program/TG-reports-for-TWG/20201130-sw-tg.md
+
+IDE - Nearing CQ. Most important issue is "hello world application".
```
Comment (jeremybennett) [inline TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:11]: More generally example applications in general.

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+The FreeRTOS PPL proposal was prepared by a team including  Jeremy Bennett, Robert Balas, Shteryana Shopova, Olive Zhao, Duncan Bees and others.
+
+In the PPL presentation by Jeremy, he noted
+FreeRTOS is already widely used by OpenHW members and is the most popular RTOS in the world
+Zephry is more widely spoken of in RISC-V world because of AntMicro – in future, we may wish to undertake support. 
+Current issues:
+* The Pulpissimo FreeRTOS work needs to be available via the legal offfice. However, if that doesn't happen, the FreeRTOS port could start from scratch.
+* Development board needs to be supplied to Shteryana – has been sent to Bulgaria
+* Simon noted that Imperas has already supported FreeRTOS for other platforms and plan to support for OpenHW when it is running on the FPGA board. So he will be added as a supporter
+ * Q? why do we need the Imperas modelling if actual boards are available?
```
Comment (jeremybennett) [inline TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:46]: General MarkDown formatting comment. Indents on lists are by increasing the asterisk characters, not by indenting. Thus this line should start `**`, similarly for other indented bullets.

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+CORE-V-VEC – Passed PPL (Nov 23). Research proposal submitted to Mitacs and feedback coming in January.
+
+CORE-V-FPGA - missed the summary - please provide, Hugh
+
+
+
+## PPL Proposals
+
+FreeRTOS
```
Comment (jeremybennett) [inline TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:34]: For clarity suggest level 3 subheading for each PPL proposal `###`

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+* André clarified that PPL and not PL is requested
+* The project will rely on GNU binutils as a starting point
+* Jeremy pointed out that LLVM linker is not ready to be used (this point needs clarification: Jeremy?)
+* The LLVM tools will support the standard instruction set and users can add support for their own custom instructions (statement needs review)
+* The project model is similar to GNU GCC tools in that it will be upstreamed to LLVM
+* The available upstream windows to hit are June 2021 or Dec 2021
+* We could upstream potentially both CV32 and CVA6 in first release milestone
+* Discussion about need for virtual customer to design initial applications - as per many of our SW work.
+* Zbigniew will get back involved in the project prior to PL
+* The PPL was approved by TWG consensus
```
Comment (jeremybennett) [inline TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:73]: For consistency with FreeRTOS, no bullet for TWG approval

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+## TG Reports
+
+### SW TG
+
+Written report prepared - thanks to Jeremy:
+https://github.com/openhwgroup/core-v-docs/blob/master/program/TG-reports-for-TWG/20201130-sw-tg.md
+
+IDE - Nearing CQ. Most important issue is "hello world application".
+
+GNU Tools - Good progress. Need to have FSF assignment from contributors - Action to Duncan.
```
Comment (jeremybennett) [inline TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:13]: Need to distinguish two actions

* OpenHW and any contributor having FSF copyright papers in place. This is a requirement for upstream contribution.  OpenHW having copyright papers in place means i) anything owned by OpenHW Group can be contributed upstream and ii) for contributors who have difficultly getting FSF copyright papers can be seconded to OpenHW Group and contribute as OpenHW Group staff.  Existing action on Duncan to get FSF copyright papers signed.

* Contributors having an agreement with OpenHW that anything they contribute to OpenHW Group repositories may subsequently be contributed upstream by OpenHW Group (giving credit to the originator).  Without this, contributions would be eligible to be upstreamed, but OpenHW Group would not have permission to contribute the work upstream.  Action on Duncan to create such an agreement for contributors to sign.

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+FreeRTOS 
+https://github.com/openhwgroup/core-v-docs/blob/master/program/core-v-free-rtos-ppl.md
+
+The FreeRTOS PPL proposal was prepared by a team including  Jeremy Bennett, Robert Balas, Shteryana Shopova, Olive Zhao, Duncan Bees and others.
+
+In the PPL presentation by Jeremy, he noted
+FreeRTOS is already widely used by OpenHW members and is the most popular RTOS in the world
+Zephry is more widely spoken of in RISC-V world because of AntMicro – in future, we may wish to undertake support. 
+Current issues:
+* The Pulpissimo FreeRTOS work needs to be available via the legal offfice. However, if that doesn't happen, the FreeRTOS port could start from scratch.
```
Comment (jeremybennett) [inline TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:43]: Offer from Rick to work with ETHZ to facilitate this.

**Review by jeremybennett — CHANGES_REQUESTED** [review]
A couple of changes to substance suggested and a couple of notes on MarkDown formatting, but otherwise good.

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+CLANG/LLVM - PPL report below
+
+FreeRTOS - PPL report below
+
+
+### HW TG
+Verbal report:
+
+CORE-V-MCU – Quicklogic is putting resources in place. Need to review against original PPL plan and schedule PL.
```
Comment (hpollittsmith) [inline TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:24]: I believe we are calling the projects CORE-V-MCU SOC and CORE-V-MCU FPGA

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+CORE-V-VEC – Passed PPL (Nov 23). Research proposal submitted to Mitacs and feedback coming in January.
+
+CORE-V-FPGA - missed the summary - please provide, Hugh
```
Comment (hpollittsmith) [inline TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:28]: CORE-V-MCU FPGA - waiting for RTL freeze for CV32E40P core to update platform. Also anticipating using FreeRTOS as the SW development environment for CORE-V-MCU FPGA and SOC.

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md (hunk 2 above)
Comment (DBees) [reply TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:46]: hmm.. saw this method on a reference but thanks.

File: TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md (hunk 5 above)
Comment (DBees) [reply TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md:13]: I wasn't putting this level of notes on the TG reports, but have now included your text as you have spelled it out so clearly.

**Pull request merged**